packages/commonwealth/client/scripts/views/components/comments/comment.tsx

```diff
@@ -108,6 +114,32 @@ export class Comment extends ClassComponent<CommentAttrs> {
       !isLocked &&
       (comment.author === app.user.activeAccount?.address || isAdminOrMod);
 
+    // if (!this.verificationChecked) {
```

Comment:
Dead code?

Reply:
Just commented out until the modal can be reviewed.